tests/core/pyspec/eth2spec/test/phase0/fork_choice/test_on_block.py

```diff
@@ -628,7 +624,7 @@ def test_new_finalized_slot_is_not_justified_checkpoint_ancestor(spec, state):
     # # Apply blocks of `another_state` to `store`
```

Comment:
Is this able to be fixed now @hwwhww ?

Reply:
Yes, fixed in https://github.com/ethereum/consensus-specs/pull/2727/commits/2c865e362799470356b703afdef83c49bb55bd38.